tests/tests.go

```diff
@@ -81,12 +81,15 @@ agent:
   topology:
     probes:
     - ovsdb
+    - blockdev
     - docker
     - lxd
     - lldp
     - runc
     - socketinfo
     - libvirt{{block "agentProbes" .}}{{"\n"}}{{range .AgentProbes}}{{println "    -" .}}{{end}}{{end}}
+    blockdev:
+      lsblk_path:
```

Comment:
This seems unnecessary (as the SKYDIVE_AGENT_TOPOLOGY_BLOCKDEV_LSBLK_PATH overrides it)